test/pytest/rbac_keycloak_users.py

```
@@ -251,6 +257,97 @@ def _list_user_realm_roles(
+
+def _find_group_id_by_name(
+    base: str,
+    realm: str,
+    admin_token: str,
+    name: str,
+) -> Optional[str]:
+    r = requests.get(
+        f"{base}/admin/realms/{realm}/groups",
+        params={"search": name, "exact": "true"},
+        headers={"Authorization": f"Bearer {admin_token}"},
+        verify=False,
```

Ruff (0.16.3)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)